web_poet/pages.py

```diff
@@ -68,6 +68,25 @@ async def to_item(self) -> ItemT:
         )
 
 
+class MultiLayoutPage(ItemPage[ItemT]):
+    """Base class for :ref:`multi-layout page object classes <multi-layout>`.
+
+    Subclasses must reimplement the :meth:`layout` method.
+    """
+
+    @abc.abstractmethod
+    async def get_layout(self) -> ItemPage[ItemT]:
```

Comment:
Not sure if this sounds right but how about `to_layout()`? In that way, we establish some sort of consistency with the PO's API naming, i.e. `to_item()`.